staging/src/k8s.io/mount-utils/mount_linux.go

```diff
@@ -48,6 +48,15 @@ const (
 	fsckErrorsUncorrected = 4
 	// Error thrown by exec cmd.Run() when process spawned by cmd.Start() completes before cmd.Wait() is called (see - k/k issue #103753)
 	errNoChildProcesses = "wait: no child processes"
+	// statCommand is a constant for statCommand
+	statCommand = `stat`
+	// statCommandArgs is the arguments that are given to the stat command. Returns device major, minor number.
+	statCommandArgs = `--format={"major": %t, "minor": %T}`
+	// statTimeout is used to time out the stat command after 300 ms. This is randomly chosen; but
+	// has been kept less than the nfsTimeout.
```

Comment:
what is the nfsTimeout?
is 300ms small?

Reply:
Apologies: 
https://github.com/kubernetes/kubernetes/blob/4ad08fa5b611826505fa2892cdce463f12d32b35/pkg/volume/nfs/nfs.go#L67

I do not think so it is small since `IsLikelyNotMountPoint` is intended to be fast.